scipy/special/tests/test_data.py

```diff
@@ -430,6 +430,8 @@ def test_boost():
         data(spherical_jn_, 'sph_bessel_data_ipp-sph_bessel_data', (0,1), 2, rtol=1e-13),
         data(spherical_yn_, 'sph_neumann_data_ipp-sph_neumann_data', (0,1), 2, rtol=8e-15),
 
+        data(owens_t, 'owenst_data_ipp-owenst', (0, 1), 2, rtol=5e-1),
```

Comment:
Are those relative errors or absolute errors? An absolute error of `1e-20` could still be bad.